css/style.css

```diff
@@ -932,10 +993,10 @@ main p {
 /* SCROLLBAR INGREDIENTS */
 
 ::-webkit-scrollbar-track {
-    box-shadow: inset 0 0 5px var(--accent); 
+    box-shadow: inset 0 0 5px var(--accent);
     border-radius: 8px;
   }
-   
```

Comment:
I think these were created/removed because I shifted my code around due to ordering, etc.

Reply:
Not changing code so no problem